src/api/app/components/workflow_artifacts_per_step_component.html.haml

```diff
@@ -0,0 +1 @@
+= artifacts_sentence(self)
```

Comment:
Since this template is only calling a single method of the view component, I suggest renaming the `artifacts_sentence` method to `call` and deleting this Haml template. It would achieve the same. Here's a reference about this from [view_component's documentation](https://viewcomponent.org/guide/templates.html#inline).

Reply:
Good advice, thanks.
I also replaced the `context` that we previously passed from the view by `helpers`.